src/esp/sensor/CameraSensor.h

```diff
@@ -245,6 +257,9 @@ class CameraSensor : public VisualSensor {
    */
   gfx::RenderCamera* renderCamera_;
 
+  CameraSensorSpec::ptr spec_ =
```

Comment:
Is this a valid way to make sure that CameraSensor has access to parameters of CameraSensorSpec, not just SensorSpec?

Reply:
I do not get your question. You already cast it to the `CameraSensorSpec`, so within this CameraSensor object, it can access any items in the CameraSensorSpec.

Reply:
Again, call it `cameraSensorSpec_` for example.

Reply:
My question was if casting was a good way to allow CameraSensor to access items in CameraSensorSpec, not just SensorSpec. (It was not cast previously) Thanks for the answer and I will rename the variable to cameraSensorSpec_!